lib/pdk/validators/ruby/rubocop.rb

```diff
@@ -11,13 +11,18 @@ def self.name
       end
 
       def self.cmd
-        'pwd'
+        'rubocop'
```

Comment:
Tried this again on a newly generated module with updated templates, and rubocop is ok with the rubocop.yml, but still finding issues with Gemfile and spec_helper.rb. I'll investigate.